packages/tsconfig-reference/scripts/tsconfigRules.ts

```diff
@@ -174,49 +174,36 @@ function trueIf(name: string) {
 }
 
 export const defaultsForOptions = {
-  allowJs: "false",
```

Comment:
Like https://github.com/microsoft/TypeScript-Website/pull/2053#discussion_r718719436, these are in sync with, e.g. `ts.optionsDeclarations.allowJs.defaultValueDescription`, but removing them ensures we pick up future updates.